src/Aspire.Hosting/Dashboard/DashboardService.cs

```diff
@@ -37,9 +38,12 @@ public override Task<ApplicationInformationResponse> GetApplicationInformation(
         ApplicationInformationRequest request,
         ServerCallContext context)
     {
+        // Read the application name from configuration if available, otherwise fall back to the environment
+        var applicationName = configuration["AppHost:DashboardApplicationName"] ?? hostEnvironment.ApplicationName;
```

Comment:
Should the value of `IHostEnvironment.ApplicationName` be improved for single file apps in general? Then all users of host environment get an improved app name, including Aspire.

Reply:
I was thinking about this, but hostEnvironment.ApplicationName is used as an assembly name in different places.